device_backends/NumericAddressedBackend/src/NumericAddressedBackend.cc

```diff
@@ -216,7 +231,7 @@ namespace ChimeraTK {
       throw ChimeraTK::runtime_error("NumericAddressedBackend is in exception state.");
     }
     catch(...) {
-      for(const auto& it : _interruptDispatchers) {
+      for(const auto& it : _primaryInterruptDispatchers) {
```

Comment:
What about the secondary interrupts? They also need to receive the exception? Are exceptions always propagated to all output interrupts of an interrupt controller?

Reply:
Yes, they have to, because the all belong to the same device. All the AsyncVariables of all interrupt controllers, no matter how deep they are nested, must see the exception according to spec.